jetson/nucleo_bridge/LCMHandler.cpp

```diff
@@ -45,23 +45,20 @@ void LCMHandler::handle_outgoing()
     // This is used as a heart beat (to make sure that nucleos do not reset, but honestly that's too difficult)
     if (NOW - last_heartbeat_output_time > heartbeat_dead_time)
     {
+        internal_object->refresh_ra_quad_angles();
         internal_object->publish_ra_pos_data();
```

Comment:
can you move publish ra_post_data & sa data into the refresh functions?

Reply:
done